gui/main.py

```diff
@@ -0,0 +1,248 @@
+import sys 
+
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
+from PyQt5.QtGui import *
+import time
+
+
+import socket
+import subprocess
+import sys
+from datetime import datetime
+import json
+
+from scanner_thread import split_processing
+
+class MainWindow(QMainWindow):
+    label = None
+    scanInProgress = False
+
+    def __init__(self, *args, **kwargs):
+        super(MainWindow, self).__init__(*args, **kwargs)
+
+        self.setWindowTitle("Port Scanner")
+        self.initUI()
+        
+        
+
+    def initUI(self):
+
+        font = QFont()
+        font.setFamily("Consolas")
```

Comment:
@Sohaib03 Just asking if this font isn't present? Which font will be the default one?

Reply:
Qt will use the font with the specified attributes, or if no matching font exists, Qt will use the closest matching installed font.